greyhounds/horace.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: greyhound
+title: Horace
+pic: horace.png
+sex: MALE
+age: 1 YEAR
+color: FAWN
+cats: 
+permalink: /greyhounds/horace/
+---
+
+More info coming soon!
```

Comment:
Horace is still missing his stuff.  

I can move adding Horace to another request so the other changes can go in.  Would you like me to do that?